src/componets/shapes/TodoListShape.js

```diff
@@ -0,0 +1,7 @@
+import PropTypes from 'prop-types';
+
+const TodoListShape = PropTypes.arrayOf(PropTypes.shape({
+  id: PropTypes.number.isRequired,
+})).isRequired;
```

Comment:
import here TodoShape and use it like this

```suggestion
const TodoListShape = PropTypes.arrayOf(TodoShape).isRequired;
```